airbyte-integrations/connector-templates/source-python/source_{{snakeCase name}}/run.py.hbs

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import sys
+
+from airbyte_cdk.entrypoint import launch
+from .source import Source{{properCase name}}
+
+if __name__ == "__main__":
```

Comment:
why is the template file different from the [connector's implementation](https://github.com/airbytehq/airbyte/pull/33414/files#diff-a27298d03056b5ce04e2d8c4aadbfa9a39771d96a5beadb01d8aa25b2ea684dfR13-R15)?

I'd expect this file to only contain the run method and for the `if __name__ == "__main__"` to be in the main.py file

Reply:
@girarda - I believe the problem is that `main.py` is external to the module (in the parent folder) and would not ship with the connector when installed as a python library. If we _move_ `main.py` into the module, then `run.py` would not be needed, per se.

In that case, then I think the entrypoint script would be:

```
"source-{{dashCase name}}=source_{{snakeCase name}}.main",
```

or 

```
"source-{{dashCase name}}=source_{{snakeCase name}}.main:main",
```

If we move `main.py`, then we'd need to update `airbyte-ci`'s `build` command and the docker image, to use something like `python -m source_apify.main ...` or (using the CLI entrypoint) `source-aplify ...` instead of invoking via `python main.py ...`.